api/proto/teleport/legacy/types/types.proto

```diff
@@ -3011,6 +3011,9 @@ message RoleOptions {
     (gogoproto.nullable) = false,
     (gogoproto.stdduration) = true
   ];
+
+  // HostUserShell is used to configure the default shell for newly provisioned host users.
+  string HostUserShell = 31 [(gogoproto.jsontag) = "host_user_shell,omitempty"];
```

Comment:
We now have third config field related to create host user feature. `CreateHostUser`, `CreateHostUserMode` and this `HostUserShell`. What do you think of creating a `CreateHostUserOptions` struct and tuck in `HostUserShell` and any new configuration options related to create host user under that?

Reply:
Got it. The third option still looks ok for now but my worry is when we need to add more of them.

If we really want to continue with this, let's update the name that matches with `create_host_user_*` pattern? Maybe `create_host_user_default_shell` so the option can be grouped together in the docs?

Reply:
Honestly `create_host_user` is redundant anyway. It would be nice if we could slowly phase it out and just have `create_host_user_mode`. Still a good point though, some kind of opts structure would keep things organized in the long run. For now I'm good with updating to `create_host_user_default_shell` :+1: